test/compute/2019-12-01/armcompute/zz_generated_virtualmachinescalesets.go

```diff
@@ -754,24 +754,24 @@ func (client *VirtualMachineScaleSetsClient) listAllHandleError(resp *azcore.Res
 	return azcore.NewResponseError(errors.New(string(body)), resp.Response)
 }
 
-// ListSKUs - Gets a list of SKUs available for your VM scale set, including the minimum and maximum VM instances allowed for each SKU.
-func (client *VirtualMachineScaleSetsClient) ListSKUs(resourceGroupName string, vmScaleSetName string, options *VirtualMachineScaleSetsListSKUsOptions) VirtualMachineScaleSetListSKUsResultPager {
-	return &virtualMachineScaleSetListSkUsResultPager{
+// ListSkus - Gets a list of SKUs available for your VM scale set, including the minimum and maximum VM instances allowed for each SKU.
```

Comment:
ListSKUs --> ListSkus...is this just coming from the swagger like this?

Reply:
Correct (unfortunately most swagger pascal-cases TLAs).  I'm testing a fix for this though.

Reply:
Fixed